rules/disallow-interpolation-for-translations.js

```diff
@@ -0,0 +1,37 @@
+const reportError = (context, node) => {
+  const reportConfig = {
+    node,
+    loc: node.loc,
+    message: `You cannot use interpolation templates for translations!`,
+  }
+  
+  const value = (node.quasi && node.quasi.quasis && node.quasi.quasis.length && node.quasi.quasis[0].value.raw) || undefined
+  if (!value) return context.report(reportConfig)
+  
+  const fix = `t("${value}")`
+  reportConfig.fix = (fixer) => {
+    return [fixer.replaceText(node, fix)]
+  }
+  
+  context.report(reportConfig)
+}
+
+module.exports = {
+  meta: {
+    docs: {
+      description: "Disallow string interpolation for translation tags",
```

Comment:
The official term is `tagged templates`, those are functions that accept a template literal as argument (see https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Template_literals)